cpu/efm32/periph/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 INRIA
```

Comment:
This file should be `rsource`d by `cpu/efm32/Kconfig`

Reply:
Upds I acutally mente to add this in another PR.